### PR TITLE
Let user configure custom zoom for tasks

### PR DESCRIPTION
**Pull request by andchiind**
This adds an endpoint where we can map an echo tag ID to a zoom value. This is not meant as a permanent solution, but it can be used until an Echo solution is created.

### CODE REVIEW

File: backend/api/Database/Models/EchoTagInspectionMetadata.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿#nullable disable
+using System.ComponentModel.DataAnnotations;
+using Api.Services.Models;
+
+namespace Api.Services.MissionLoaders
+{
+    public class EchoTagInspectionMetadata
```
Comment (Christdej) [inline backend/api/Database/Models/EchoTagInspectionMetadata.cs:7]:
```suggestion
    public class TagInspectionMetadata
```

File: backend/api/Database/Context/FlotillaDbContext.cs [hunk 2]
```diff
@@ -26,6 +27,7 @@ public FlotillaDbContext(DbContextOptions options) : base(options)
         public DbSet<DefaultLocalizationPose> DefaultLocalizationPoses => Set<DefaultLocalizationPose>();
         public DbSet<AccessRole> AccessRoles => Set<AccessRole>();
         public DbSet<UserInfo> UserInfos => Set<UserInfo>();
+        public DbSet<EchoTagInspectionMetadata> EchoTagInspectionMetadata => Set<EchoTagInspectionMetadata>();
```
Comment (Christdej) [inline backend/api/Database/Context/FlotillaDbContext.cs:30]: Avoid using Echo naming

File: backend/api/Controllers/EchoController.cs
Comment (Christdej) [inline backend/api/Controllers/EchoController.cs:1]: Change to be a "Inspection" or "InspectionMetadata" controller, dont want to lock into Echo naming

**Review by Christdej — CHANGES_REQUESTED** [review]

File: backend/api/Database/Context/FlotillaDbContext.cs (hunk 2 above)
Comment (andchiind) [reply backend/api/Database/Context/FlotillaDbContext.cs:30]: This object is linked to the EchoTag model which already exists in the codebase

Comment (Christdej) [reply backend/api/Database/Context/FlotillaDbContext.cs:30]: Not in the database, only in the service and controller models

**Review by Christdej — APPROVED** [review]
LGTM

**Pull request merged**